From c7bec3320f6907c4edd8416a5e5cfa89b071a411 Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Wed, 13 Jul 2016 15:36:09 +0200
Subject: [PATCH] tm: execut cfg_update() inside t_continue()

- refresh the values changed inside cfg framework
- reported by Victor Seva, GH #708
---
 modules/tm/t_suspend.c | 3 +++
 1 file changed, 3 insertions(+)

--- a/modules/tm/t_suspend.c
+++ b/modules/tm/t_suspend.c
@@ -22,6 +22,7 @@
 #include "../../action.h"
 #include "../../script_cb.h"
 #include "../../dset.h"
+#include "../../cfg/cfg_struct.h"
 
 #include "config.h"
 #include "sip_msg.h"
@@ -177,6 +178,8 @@
 	int do_put_on_wait;
 	struct hdr_field *hdr, *prev = 0, *tmp = 0;
 
+	cfg_update();
+
 	if (t_lookup_ident(&t, hash_index, label) < 0) {
 		LM_ERR("transaction not found\n");
 		return -1;
